users/vieting/experiments/switchboard/ctc/feat/experiments.py

```diff
@@ -209,7 +238,6 @@ def run_scf_baseline():
         "datasets": returnn_datasets,
         "extra_args": {
             "accum_grad_multiple_step": 2,
-            "watch_memory": True,
```

Comment:
Why do you remove this?

Reply:
I thought it was mostly for debugging purposes and is no longer needed.

Reply:
I added it back again for the SCF baseline since it broke my hash. I will leave it unchanged for now